### PR TITLE
Refactor/view component params and remove db query

### PR DESCRIPTION
**Pull request by wwrk22**
## What's the change?
Refactor for:
- Consistent namespacing for view components and pair request policies
- Clean up `StandupMeetings::CheckInActionComponent` to help improve readability and manageability of `app/views/standup_meetings/index.html.erb` in particular, so that it may be easier to add new view components to the template.
- Move a couple of database queries being run by a view component to appropriate places.

## What key workflows are impacted?
N/A

## Highlights / Surprises / Risks / Cleanup
N/A

## Demo / Screenshots
N/A

## Issue ticket number and link
N/A

## Checklist before requesting a review

Please delete items that are not relevant.

- [x] Did you add appropriate automated tests?
- [x] Did you consider risks around security, performance, etc.?
- [x] Have you thought of misfiring code? e.g. too many loops, n+1, or how to handle nils?
- [x] Any validations to data needed?
- [x] Related to readability and consistency: Did you add comments and/or documentation? (README, Notion, etc.)
- [x] Is everything elevated to the highest level? (e.g., can logic be moved out of view into controllers, or out of controllers into models?)
- [x] Can (and should) any models be extracted?
- [x] Is there any linting that needs to be executed?
- [ ] Did you leave helpful inline PR comments for the reviewer(s)?
- [x] Did you include instructions for how to test in the ticket?
- [x] Did you add any relevant tags and decide if this PR is a Draft vs. Ready for Review?
- [x] Do you have a plan for how this change should be rolled back? (e.g., how do you deal with a migration rollback? Is your feature released behind a feature flag?)

### CODE REVIEW

File: app/components/standup_meetings/check_in_action_component.html.erb
Comment (wwrk22) [inline app/components/standup_meetings/check_in_action_component.html.erb:1]: Compare to `app/components/standup_meeting/check_in_action_compnent.html.erb` for review.

File: app/components/standup_meetings/check_in_action_component.rb
Comment (wwrk22) [inline app/components/standup_meetings/check_in_action_component.rb:1]: Compare to `app/components/standup_meeting/check_in_action_compnent.rb` for review (`standup_meeting` singular).

File: app/controllers/standup_meetings_controller.rb [hunk 1]
```diff
@@ -8,12 +8,18 @@ def index
         date: Date.current.strftime('%m-%d-%Y'))
     end
 
-    @standup_meeting_group = StandupMeetingGroup.includes(:users)
```
Comment (wwrk22) [inline app/controllers/standup_meetings_controller.rb:11]: There is no call to `user` on `@standup_meeting_group` throughout its variable life as far as I can tell. Please let me know if I missed something. All existing tests are passing and I’ve verified manually by running the app that the change hasn’t broken anything.

File: app/controllers/standup_meeting_groups/joins_controller.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class StandupMeetingGroups::JoinsController < ApplicationController
   # POST /standup_meeting_groups/:standup_meeting_group_id/joins.turbo_stream
   def create
-    @standup_meeting_group = StandupMeetingGroup.find(params[:standup_meeting_group_id])
+    @standup_meeting_group = StandupMeetingGroup.includes(:standup_meetings)
```
Comment (wwrk22) [inline app/controllers/standup_meeting_groups/joins_controller.rb:4]: There are a couple of view components that call `standup_meetings` on `@standup_meeting_group`, which would trigger database queries without this `includes`.

**Review by JoshDevHub — APPROVED** [review]
LGTM :rocket:

**Pull request merged**